shared/provision/error/container.tsx

```diff
@@ -25,3 +25,10 @@ export default connect(
     onAccountReset: () => d._onAccountReset(s._username),
   })
 )(RenderError)
+
+// @ts-ignore
+ConnectedRenderError.navigationOptions = {
```

Comment:
you should put these types of things inside the actual component. connect will hoist correctly